README.md

```diff
@@ -0,0 +1 @@
+# ButterCMS Alexa Skill Example
```

Comment:
Can you add some high level getting started steps here for other folks that want to try using this skill?

Reply:
Ok, I will add some info on the README!

Reply:
Thanks - please let me know once it's ready and I'll take a final look.

Reply:
@jakelumetta done!